### PR TITLE
Add docs about reportSnabbdomError

### PR DESCRIPTION
**Pull request by ryota-ka**
- [ ] There exists an issue discussing the need for this PR
- [ ] I added new tests for the issue I fixed or built
- [x] I used `pnpm run commit` instead of `git commit`

This PR is a successor of https://github.com/cyclejs/cyclejs/pull/923
I've updated docs to note about `reportSnabbdomError`.

### CODE REVIEW

**Comment by staltz** [comment]
@ryota-ka Please notice the instructions in this readme: https://github.com/cyclejs/cyclejs/blob/master/docs/README.md

**Comment by ryota-ka** [comment]
@staltz oops, sorry!
However, I've noticed that `docs/content/api/dom.md` has only 41 lines, and we miss whole of the latter part we have in `docs/api/dom.html`.
(The reason I edited HTML directly is I couldn't find `DOMDriverOptions` with `$ git grep` in `docs/content`.)

Am I missing something?

**Comment by jvanbruegge** [comment]
Here are the docs for the arguments: https://github.com/cyclejs/cyclejs/blob/master/dom/src/index.ts

**Comment by ryota-ka** [comment]
@jvanbruegge Thanks for your advice.
It's quite hard to notice that...

I updated `{dom,html}/src/index.ts` and ran `pnpm run docs`, but I don't see any changes under `docs/api`.
Am I still missing somethig? Or will @staltz reflect changes to HTML files just before the release of  the new version?

Anyway, I've force-pushed my changes.

File: html/src/index.ts [hunk 1]
```diff
@@ -40,8 +40,9 @@ export {HTMLSource} from './HTMLSource';
  * @param {Function} effect a callback function that takes a string of rendered
  * HTML as input and should run a side effect, returning nothing.
  * @param {HTMLDriverOptions} options an object with one optional property:
```
Comment (staltz) [inline html/src/index.ts]: @ryota-ka Could this line say "options an object with **two** optional properties:" ?

**Comment by staltz** [comment]
@ryota-ka Thanks for your patience. The docs generation is not the nicest part of our monorepo

> I updated `{dom,html}/src/index.ts` and ran `pnpm run docs`, but I don't see any changes under `docs/api`.

I think you have to build (i.e. typescript compile) before `run docs`.

> Am I still missing somethig? Or will @staltz reflect changes to HTML files just before the release of  the new version?

I will regenerate the HTML, yes.

File: html/src/index.ts (hunk 1 above)
Comment (ryota-ka) [reply html/src/index.ts]: oops, I missed this when I edited this file. thanks!

**Comment by ryota-ka** [comment]
> The docs generation is not the nicest part of our monorepo

Then could I make another pull request to update `docs/README.md` to make it easy for other people to contribute to this project?

---

> > Am I still missing somethig? Or will @staltz reflect changes to HTML files just before the release of  the new version?
> 
> I will regenerate the HTML, yes.

Then is it enough for me to edit `{dom,html}/src/index.ts` in this PR?
If so, please take another look 🙏

**Comment by staltz** [comment]
Thank you @ryota-ka !

**Comment by staltz** [comment]
Website is updated

**Pull request merged**